src/parliament/views.py

```diff
@@ -12,7 +12,14 @@
 @api_view(['GET', 'POST'])
 @permission_classes((IsAuthenticatedOrReadOnly,))
 def parliament_list(request):
+    '''
+    get:
+        Get a list of all parliments.
 
+    post:
+        Add new parliment with a given ID.
```

Comment:
You are correct, I must have thought the endpoint was "POST: /api/v1/parliaments/{id}"